records_mover/check_db_conn_engine.py

```diff
@@ -0,0 +1,31 @@
+# flake8: noqa
+
+import sqlalchemy
+from typing import Union, Optional, Tuple
+import warnings
+
+
+def check_db_conn_engine(db: Optional[Union[sqlalchemy.engine.Engine,
```

Comment:
this is a nicely encapsulated function that could definitely benefit from unit tests for each of its branches

Reply:
Great point, added here: https://github.com/bluelabsio/records-mover/blob/f22acfd664b99283ba8dadf07528aabf5d93bf17/tests/unit/test_check_db_conn_engine.py